packages/shader-lab/src/common/BaseScanner.ts

```diff
@@ -35,13 +39,15 @@ export default class BaseScanner {
   }
 
   get curPosition(): ShaderPosition {
-    return ShaderLab.createPosition(
-      this._currentIndex,
-      // #if _EDITOR
-      this._column,
-      this._line
-      // #endif
-    );
+    return ShaderLab.createPosition(this._currentIndex, this._line, this._column);
```

Comment:
@![image](https://github.com/user-attachments/assets/fee7bd92-426e-4de3-942b-e9503beeff5c)

`get curPosition` ->`createPosition()` is better!
```ts
 get curPosition(): ShaderPosition {
    return ShaderLab.createPosition(this._currentIndex, this._line, this._column);
  }
```

Reply:
done